ktlint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Run ktlint but exclude rules that are currently not applied to this project.
+
+if ./ktlint | grep -Ev "(Wildcard import|Unexpected spacing before)"; then
+  echo "Please correct above lint issues."
+  exit 1
+else
+  echo "No relevant lint issues found."
```

Comment:
While we're at it, should we say "ktlint issues" here (and also above) to distinguish from lint issues found by detekt?

Reply:
Done.